File: Udemy/Algoritmos e Logica de Programacao/C/Estrutura Sequencial/ExerciciosEstruturaSequencial/ProblemaDuracao.c
/*Problema "duracao" Fazer um programa para ler uma duração de tempo em segundos,
daí imprimir na tela esta duração no formato horas:minutos:segundos.*/


#include<stdio.h>
#include<stdlib.h>

int main(){

    int duracao, hora, minutos, segundos, resto;

    printf("Digite a duracao em segundos: ");
    scanf("%d",&duracao);

    hora = duracao / 3600;
    resto = duracao % 3600;

    minutos = resto / 60;
    segundos = resto % 60;

    printf("%d:%d:%d",hora,minutos,segundos);


    return 0;
}

File: Algoritmos e Logica de Programacao/Java/ExerciciosJavaEstruturaSequencial/src/application/ProblemaDuracao.java
package application;

import java.util.Locale;
import java.util.Scanner;

public class ProblemaDuracao {

	public static void main(String[] args) {
		/*
		 * Problema "duracao"
Fazer um programa para ler uma duração de tempo em segundos, daí imprimir na tela esta duração no
formato horas:minutos:segundos.
		 * */
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int duracao, horas, minutos, segundos, resto;
		
		System.out.print("Digite a duracao em segundos: ");
		duracao = sc.nextInt();
		
		horas = duracao / 3600;
		resto = duracao % 3600;
		minutos = resto / 60;
		segundos = resto % 60;
		
		System.out.println(horas+":"+minutos+":"+segundos);
		
		sc.close();
	}

}
